src/data-workspace/data-workspace.js

```diff
@@ -29,17 +27,8 @@ export const DataWorkspace = ({ selectionHasNoFormMessage }) => {
     const { data: metadata } = useMetadata()
     useHandleHeaderbarStatus()
     useCheckLockStatus()
-    const { lockStatus: frontEndLockStatus, setLockStatus } = useLockedContext()
     const updateStore = useValueStore((state) => state.setDataValueSet)
-    const initialDataValuesFetch = useDataValueSet({
-        onSuccess: (data) => {
-            updateLockStatusFromBackend(
-                frontEndLockStatus,
-                data?.lockStatus,
-                setLockStatus
-            )
-        },
-    })
+    const initialDataValuesFetch = useDataValueSet()
```

Comment:
Since `useCheckLockStatus` uses a `useEffect` dependency on `dataValueSet.data?.lockStatus`, the update from the backend resulted in two updates -- see also `use-check-lock-status.js`